gztextractor/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+GZ Text Extractor - LLM-based Sri Lankan Government Gazette Data Extraction Tool
+
+This package provides tools for extracting structured data from Sri Lankan Government Gazette PDFs.
+"""
+
+from .main import run_pipeline
+
+__version__ = "1.0.0"
+__author__ = "Isuru Rangana"
```

Comment:
@Isuru-rangana we shouldn't put personal names, this should be Lanka Data Foundation. But everyone knows you wrote this code as the commit history has it.